src/components/layout/layout.js

```diff
@@ -69,9 +75,11 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
 
                 if (!response.error) {
                     setClientCountry(response.website_status.clients_country)
-                    Cookies.set('clients_country', response.website_status.clients_country, {
-                        expires: cookie_expires,
-                    })
+                    clients_country_cookie.set(
+                        CLIENTS_COUNTRY_KEY,
+                        response.website_status.clients_country,
+                        { sameSite: 'none', secure: true },
```

Comment:
can we enforce samsite none and secure true by default, we can move this to storage.js cookie setup instead?

Reply:
Sure.